### ISSUE TITLE
Related parent - child not processed

### ISSUE THREAD

**Issue by MaRRiK74** [leonardoporro/Detached-Mapper#43]
Sorry to bother again... This is kind of a follow-up of #41.

I am using a generic repository, when updating parent/child like in issue 41 that seems to work. But when I try to update a parent/child (parent is new / child exists). The relation on the child is not saved to DB.

This is my AddAsync with a dbresult to check for the result in DB:
![image](https://user-images.githubusercontent.com/19833001/141662512-d2d6aecc-83fa-4dbb-a59f-6023380740a7.png)

After saving the changes this is the result:
![image](https://user-images.githubusercontent.com/19833001/141662562-549effce-28f4-4007-9a3d-dd1428984cb4.png)

Variables:
obj = object which should be persisted
dbresult = result of db query after saving
result = mapper result

**Comment by MaRRiK74** [comment]
Additional info:

![image](https://user-images.githubusercontent.com/19833001/141662663-b72c5cb4-cf9b-47bb-8a46-e96fe0ac42d5.png)

BaseModel:

![image](https://user-images.githubusercontent.com/19833001/141662691-ea8ee1b2-00c9-4f2e-b1c2-be523ec9f320.png)

**Comment by leonardoporro** [comment]
mm.. is that many to many? could you add the [Aggregation] or [Composition] attribute? (or define it using config in UseDetached).
Would it be possible to extract your sample to a test or console app or something that can be attached here or sent it by email?

**Comment by MaRRiK74** [comment]
Yes it is a many to many relation.

I have created a small console app, and guess what, it works as expected. Then I went back fired up the 'old' project, nothing changed there, but now it does return the correct results. Blows my mind..

Maybe VIsual Studio 2022 was the culprit. The last few days I've had some weird behavior with it, like not loading a project correctly and throwing errors. Restarting VS2022 and reloading the project solved that problem too.
